attributes/default.rb

```diff
@@ -19,3 +19,12 @@
 #
 
 default['ms_dotnet']['timeout'] = 600
+
+if platform? 'windows'
+  nt_version = node['platform_version'].to_f
+  if nt_version >= 6.2
+    default['ms_dotnet']['feature']['enable_all']            = true
```

Comment:
Hi @Annih , I thought about the 'v3' in the attribute namespace before my commit. I just thought that this is a functionality is conditioned based on the windows platform and not .Net version. Therefore the functionality would apply to any .Net version on nt_version 6.2 or higher. What do you think?